cmd/subnetcmd/stats.go

```diff
@@ -183,7 +289,7 @@ func findAPIEndpoint(network models.Network) (platformvm.Client, info.Client) {
 	_, err := c.GetHeight(ctx)
 	if err == nil {
 		i = info.NewClient(constants.LocalAPIEndpoint)
-		_, err := i.GetNodeID(ctx)
+		_, _, err := i.GetNodeID(ctx)
```

Comment:
Is this just checking to make sure the client works? Maybe add a comment to explain

Reply:
yep that's what it's doing. Ok adding comment.